rules/cre/snvreport_mosaic.smk

```diff
@@ -0,0 +1,34 @@
+#callers = [ "freebayes_mosaic" ]
+
+rule allsnvreport_mosaic:
```

Comment:
Why are there two snakemake files for the mosaic report? i.e. this one also snvrepor_mosaic.smk?

Reply:
I deleted the allsnvreport_mosaic.smk file :) The snvreport_mosaic.smk is the file renamed from allsnvreport_mosaic.smk